app/models/guest_application.rb

```diff
@@ -1,6 +1,7 @@
 class GuestApplication < ActiveRecord::Base
   belongs_to :vm_or_template
   belongs_to :host
+  belongs_to :container_image
```

Comment:
is it ok there's both belongs_to host and belongs_to container_image? shouldn't it be only one of them at a certain time?